infrastructure/main.tf

```diff
@@ -1,5 +1,16 @@
-provider "azurerm" {
-  version = "1.44.0"
+terraform {
```

Comment:
might cause a problem you already have this defined in state.tf

https://github.com/hmcts/draft-store/blob/master/infrastructure/state.tf#L1

Reply:
👍